src/elements/content-sidebar/activity-feed/activity-feed/ActiveState.js

```diff
@@ -170,7 +170,9 @@ const ActiveState = ({
                                     currentUser={currentUser}
                                     getAvatarUrl={getAvatarUrl}
                                     getUserProfileUrl={getUserProfileUrl}
+                                    error={item.error}
                                     isCurrentVersion={currentFileVersionId === itemFileVersionId}
+                                    isPending={item.isPending}
                                     item={item}
```

Comment:
Since `item` is available here, can we pull `error` and `isPending` out of it inside the component?